src/content/basics/giant-swarm-operational-layers/index.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Giant Swarm Operational Layers"
+description = "Here you learn how the operational layers of Giant Swarm are defined and what the intended operational model is."
+date = "2018-10-12"
+type = "page"
+weight = 30
+categories = ["basics"]
++++
+
+# Giant Swarm Operational Layers
+
+A Giant Swarm installation has several operational layers, which depict a separation of concerns both on an operational as well as on a security level. In the following we will define the layers and explain the intended operational model.
+
+## Operational Layers
+
+We will go through the operational layers one by one from the bottom (infrastructre) to the top (user space) and explain the intended opreational model by defining (typical) users and permission levels. The layers are:
+
+1. [Infrastructure](#infrastructre)
+2. [Giant Swarm Control Plane](#controlplane)
+3. [Giant Swarm API](#GSAPI)
+4. [User Space](#userspace)
+
+### Infrastructure {#infrastructre}
+
+The infrastructre layer covers the area on top of actual (or virtual) machines, networking, etc., which is managed by Giant Swarm SREs usually accessed through VPN and bastion hosts as well as through the Cloud Provider APIs if applicable.
```

Comment:
Is `SREs` a common term by now? If not, I'd either add the meaning in brackets, or replace it by `staff`.

Reply:
will add